benchmarks/315.sysbench/sysbench.sh

```diff
@@ -53,103 +106,62 @@ if [ ! -d $sysbench_dir ]; then
     fi
 fi
 
-mysql_loc=($(whereis mysql))
-
-OSname_Ubuntu=`cat /etc/*release | grep -c "Ubuntu"`
-OSname_CentOS=`cat /etc/*release | grep -c "CentOS"`
-if [ ! $OSname_Ubuntu -eq 0 ]; then
-    for i in ${mysql_loc[@]}; do
-        tmp=$(echo $i | grep '/lib/mysql')
-        if [ $? -eq 0 ]; then
-            mysql_lib=$tmp
-        else
-            tmp1=$(echo $i | grep '/include/mysql')
-            if [ $? -eq 0 ]; then
-                mysql_include=$tmp1
-            fi
-        fi
-    done
-elif [ ! $OSname_CentOS -eq 0 ]; then
-    for i in ${mysql_loc[@]}; do
-        grep -q '/lib[0-9]*/mysql' <<< "${i}"
-        if [ $? -eq 0 ]; then
-             mysql_lib=${i}
-        else
-            grep -q '/include/mysql' <<< "${i}"
-            if [ $? -eq 0 ]; then
-                mysql_include=${i}
-            fi
-        fi
-    done
-else
-    echo "selected OS in not Ubuntu or centos"
-    exit 1
-fi
-
-if [ "$mysql_lib" == "" -o "$mysql_include" == "" ]; then
-    echo 'mysql has not been installed right'
-    exit 1
-fi
-
-if [ ! -d $sysbench_dir ]; then
-    echo 'sysbench has not been download completely'
-    exit 1
-fi
-
-export PATH=$PATH:/usr/local
-
+#sysbench compile
 pushd $sysbench_dir
   prefix=/usr/local/sysbench
   ./autogen.sh
-  ./configure --prefix=$prefix --with-mysql-includes=$mysql_include --with-mysql-libs=$mysql_lib
+  ./configure --prefix=$prefix --with-mysql-includes=$drIncMysql --with-mysql-libs=$drLibMysql
   make -s 
   make install
 popd
 
-/usr/bin/expect <<EOF
-set timeout 40
-
-spawn mysql -u$mysql_user -p
-expect "*password:"
-send "$mysql_password\r"
-expect "mysql>"
-send "show databases;\r"
-
-expect {
- "$db_name"
- {
-     send "drop database $db_name;\r"
-     expect "mysql>"
-     send "create database $db_name;\r"
-     expect "mysql>"
- }
- "mysql>"
- {
-     send "create database $db_name;\r"
- }
-}
-expect "mysql>"
-send "quit;\r"
-expect eof
-EOF
```

Comment:
Why didi you drop these operations for database? If dropped, the next test in the same board will lead error.